lonestar/liblonestar/include/Lonestar/Utils.h

```diff
@@ -36,8 +37,10 @@ MakeFileGraph(
     edge_properties.emplace_back(edge_property_name);
   }
 
-  auto pfg_result =
-      katana::PropertyGraph::Make(rdg_name, node_properties, edge_properties);
+  tsuba::RDGLoadOptions opts;
+  opts.node_props = &node_properties;
+  opts.edge_props = &edge_properties;
+  auto pfg_result = katana::PropertyGraph::Make(rdg_name, opts);
```

Comment:
Use `{.node_props = ...}` syntax as above?

Reply:
If you don't directly control the class, `{.foo = ...}` style initialization is a little brittle. The initialization order needs to be in member definition order and depending on your compiler version, you have to initialize all the fields, which puts a damper on adding options without have to change callers.

Reply:
<rant>OMG. Why is C++ such a POS?!?!?!?! I thought they finally had a sane way to do named parameters and defaults where ordering doesn't matter. </rant>

OK. Never mind by recommendations then.